src/screens/UserManagement/UserManagementTypes.res

```diff
@@ -0,0 +1,45 @@
+type permissionType =
+  | OperationsView
+  | OperationsManage
+  | ConnectorsView
+  | ConnectorsManage
+  | WorkflowsView
+  | WorkflowsManage
+  | AnalyticsView
+  | UsersView
+  | UsersManage
+  | MerchantDetailsView
+  | MerchantDetailsManage
+  | OrganizationManage
+  | UnknownPermission(string)
+
+open AuthTypes
+type permissionJson = {
+  operationsView: authorization,
+  operationsManage: authorization,
+  connectorsView: authorization,
+  connectorsManage: authorization,
+  workflowsView: authorization,
+  workflowsManage: authorization,
+  analyticsView: authorization,
+  usersView: authorization,
+  usersManage: authorization,
+  merchantDetailsView: authorization,
+  merchantDetailsManage: authorization,
+  organizationManage: authorization,
+}
+
+// TODO : Remove this type definition as no longer used
+type permissions = {
+  description: string,
+  enum_name: string,
+  mutable isPermissionAllowed: bool,
```

Comment:
remove isPermissionAllowed type definition it not required

Reply:
Will refactor and remove unused type definitions in the next PR .

Reply:
https://github.com/juspay/hyperswitch-control-center/pull/471#discussion_r1510867960